grails-app/controllers/com/unifina/controller/api/LoginApiController.groovy

```diff
@@ -53,7 +53,7 @@ class LoginApiController {
 
 	@StreamrApi(authenticationLevel = AuthLevel.NONE)
 	def apikey(ApiKeyCommand cmd) {
-		SecUser user = userService.getUserFromApiKey(cmd.apiKey)
+		SecUser user = userService.getUserFromApiKey(cmd.apikey)
```

Comment:
I just realized that in `streamr-client` we use `apiKey`, right? Would be good to be consistent, also all-lower-caps looks kinda funny. Would it be a nightmare to change it back? I apologize if I earlier gave the opposite advice!

Reply:
@mthambipillai Maybe we're looking at some different places, but for the config option you give to `StreamrClient` is `apiKey`. See e.g. [this line](https://github.com/streamr-dev/streamr-client/blob/master/src/StreamrClient.js#L25)?